spec/like_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  subject { Like.new(user_id: 1, post_id: 1) }
+
+  before { subject.save }
+
+  it 'new like should be saved in the database' do
```

Comment:
## * Kindly Add unit specs for all of your models' methods as unit tests for them are missing ( for example you have the method `update_post_likes_counter` which needs to be tested ) 
![image](https://github.com/torobucci/blog_app/assets/52497246/8955219d-0c4f-41dc-bf1a-ed6ade4df6db)